scripts/run-all-checks.sh

```
@@ -412,6 +413,27 @@ run_code_checks() {
+        # change the behaviour of the stable rules and raise thousands of new
+        # findings. Naming the codes explicitly keeps the surface narrow.
+        #
+        # Only the three rules that fire on code are selected. E114, E115 and
+        # E116 are their comment-line counterparts, and E117 fires on both.
+        # Where a comment sits is the author's decision here: a trailing comment
+        # continued under the column it started in, an annotation that follows
+        # the statement it describes, and commented-out code parked at column 0
+        # are all deliberate, and all three read worse pulled to the code grid.
+        print_info "Running ruff check (indentation)..."
+        if python -m ruff check --preview \
+                --select E111,E112,E113 $RUFF_TARGETS; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)